sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/OfferAccepter.java

```diff
@@ -54,6 +55,14 @@ public OfferAccepter(List<OperationRecorder> recorders) {
             LOGGER.warn("No Operations to perform.");
         }
 
+        launchCount.inc(
```

Comment:
this could be done in an `OperationRecorder` implementation, which would be invoked as part of the `record()` call at L46.

Reply:
Could just record all operations (not just LAUNCH and LAUNCH_GROUP). This could be done by getting the `Operation.Type`, lowercasing it, and using that as the metric name.